### PR TITLE
refactor(svelte): Improve integration with `sg`

### PR DESCRIPTION
**Pull request by fkling**
There currently isn't a 'one step' way to start a local Sourcegraph instance with the SvelteKit app.

This commit adds `sg start enterprise-sveltekit` to fix that.

The changes in https://github.com/sourcegraph/sourcegraph/pull/64272 allow us to run the vite dev server in front of the Sourcegraph instance, instead of building the assets and having them served by frontend.
This gives us the benefit of hot module reloading and in general seems to be a less fragile approach.

It's basically the same what we do with the React app in development mode.

## Test plan

`sg start enterprise-sveltekit` starts the vite dev server as well as the sourcegraph instance. Navigating to `https://sourcegraph.test:3443/search` opens the Svelte app (when enabled and logged in). Making a change in a source file updates the web page immediately.

`sg start web-sveltekit-standalone` still works

### CODE REVIEW

File: sg.config.yaml [hunk 1]
```diff
@@ -564,13 +571,6 @@ commands:
       # place in standalone mode.
       DEPLOY_TYPE: ""
 
-  web-sveltekit-prod-watch:
-    description: Builds the prod version of the SvelteKit web app and rebuilds on changes
-    cmd: pnpm --filter @sourcegraph/web-sveltekit run build --watch
-    install: |
-      pnpm install
-      pnpm generate
-
```
Comment (fkling) [inline sg.config.yaml:573]: I don't think this is necessary or was used a lot.

File: sg.config.yaml [hunk 2]
```diff
@@ -1547,6 +1532,38 @@ commandsets:
...
+      - zoekt-web-0
+      - zoekt-web-1
+      - blobstore
+      - embeddings
+    env:
+      DISABLE_CODE_INSIGHTS_HISTORICAL: false
+      DISABLE_CODE_INSIGHTS: false
+      CADDY_FILE: dev/Caddyfile-sveltekit
+      # Proxy to the local server
+      SOURCEGRAPH_API_URL: http://localhost:3080
```
Comment (fkling) [inline sg.config.yaml:1565]: I wish there was a better way to just add one command to the existing command set :-/

**Review by camdencheek — APPROVED** [review]
This looks great -- thank you!

I would really love it if we could get `sg start` to "just work" as well (including both react and svelte versions of the webapp). Even if HMR doesn't work, it would be nice to have that better represent a real instance.

**Comment by fkling** [comment]
> I would really love it if we could get sg start to "just work" as well (including both react and svelte versions of the webapp).

Yeah. We could certainly just run `pnpm build --watch`. At the same time rebuilding seems to be very slow (in general the build is quite slow) so developing the SvelteKit app with it is not going to be pleasant.

**Pull request merged**